src/vnsw/agent/contrail/SConscript

```diff
@@ -89,23 +88,23 @@ env.Depends('main.o', 'buildinfo.cc')
 # Add contrail-vrouter-agent specific libraries
 env.Prepend(LIBS = AgentEnv['CONTRAIL_LIBS'])
 
-# Whole-arhive flag causes the library to be included in the resulting output
-# even though there may not be any calls requiring its presence. We need this
-# for kstate library so that the HTML file for kstate  is listed in agent
-# introspect page
-#env.Prepend(LINKFLAGS = ['-Wl,--whole-archive', '-rdynamic', '-lkstate', '-ldiag',
-#                         '-Wl,--no-whole-archive'])
-env.Prepend(LINKFLAGS = 'Iphlpapi.lib')
+if not sys.platform.startswith('win'):
```

Comment:
Didn't we decide to use sys.platform != 'win32' instead of "startswith"?

Reply:
Generally yes but in this file `.startswith` is heavily used, so I decided to just go with it.